src/misc_gui.cpp

```diff
@@ -139,42 +139,11 @@ class LandInfoWindow : public Window {
 	{
 		Town *t = ClosestTownFromTile(tile, _settings_game.economy.dist_local_authority);
 
-		/* Because build_date is not set yet in every TileDesc, we make sure it is empty */
-		TileDesc td;
-
-		td.build_date = CalendarTime::INVALID_DATE;
-
-		/* Most tiles have only one owner, but
-		 *  - drivethrough roadstops can be build on town owned roads (up to 2 owners) and
-		 *  - roads can have up to four owners (railroad, road, tram, 3rd-roadtype "highway").
-		 */
-		td.owner_type[0] = STR_LAND_AREA_INFORMATION_OWNER; // At least one owner is displayed, though it might be "N/A".
```

Comment:
Either set this single value here, in the constructor or in all callers that do not override it.